jobs/haproxy/spec

```diff
@@ -114,3 +118,28 @@ properties:
           - 10.20.10.11
           backend_port: 80 # optional - sets backend port - otherwise defaults to `port`
           ssl: true        # optional - enables ssl, and uses the `ha_proxy.ssl_pem` provided key
+
+  keepalived.vip:
+    description: VIP that will be given to master
+  keepalived.healthcheck_name:
+    description: name of healthcheck that will be performed
+    default: check_haproxy
+  keepalived.healthcheck_command:
+    description: check that will be performed
+    default: killall -0 haproxy
+  keepalived.interface:
+    description: interface keepalived will use to mount the VIP
+    default: eth0
+  keepalived.masterip:
+    description: IP of masternode
+  keepalived.vrrp_instance_name:
+    description: name of the VRRP instance
+  keepalived.master_priority:
+    description: master weight
+    default: 101
+  keepalived.slave_priority:
+    description: slave weight
+    default: 100
+  keepalived.lvs_id:
+    description : Keepalived process identifier
```

Comment:
The default value (possibly assigned by the erb template instead of a spec file default value) needs to be as unique as possible to avoid collisions with other instances of cf-haproxy-boshrelease for 
- two AZs of a same bosh "mysql" service deployment
- two bosh deployments (e.g. mysql and redis) deployments
- the same deployment in distinct bosh directors

Looking at https://bosh.io/docs/jobs.html it's not clear the default properties ``<%= spec.* %>`` available to templates is sufficient/available.